src/test/java/org/broadinstitute/hellbender/engine/FeatureSupportIntegrationTest.java

```diff
@@ -83,6 +92,25 @@ public void testFeaturesAsIntervalsUnrecognizedFormatFile() throws IOException {
         testSpec.executeTest("testFeaturesAsIntervals", this);
     }
 
+    @Test
+    public void testSitesOnlyMode() {
```

Comment:
Can you move this to a new `GATKToolIntegrationTest` class instead? I regret my original suggestion of `FeatureSupportIntegrationTest`